clients/src/main/java/org/apache/kafka/common/requests/ProduceRequest.java

```diff
@@ -40,8 +40,9 @@
 import static org.apache.kafka.common.requests.ProduceResponse.INVALID_OFFSET;
 
 public class ProduceRequest extends AbstractRequest {
+    public static final short LAST_BEFORE_TRANSACTION_V2_VERSION = 11;
```

Comment:
nit: can we rename this to LAST_STABLE_VERSION_BEFORE_TRANSACTION_V2 to be consistent with the EndTxnRequest changes we made recently